### PR TITLE
enhance(p6) to clarify message class structure

### PR DESCRIPTION
**Pull request by leesjensen**
Better represent the CONNECT message and how to implement the WebSocket commands in phase 6.

### CODE REVIEW

**Review by 19mdavenport — APPROVED** [review]
Looks good to me!

File: chess/6-gameplay/gameplay.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ classDiagram
 
 ```
 
-The important thing is that along with the fields that are generally required for a `UserGameCommand`, that there is also an additional field named `move` that contains the fields of a `ChessMove` object in your JSON serialization.
+In the end, the important thing is that you design supports the additional `move` field when serializing the `MAKE_MOVE` command over the WebSocket. This must looks something like the following:
```
Comment (frozenfrank) [inline chess/6-gameplay/gameplay.md:107]: This is great!

There are two adjustments:
* Fix "you design" to "your design"
* Fix "This must looks something like..."
    * Possibility 1: "This looks something like..."
    * Possibility 2: "This must look like..."

File: chess/6-gameplay/gameplay.md [hunk 2]
```diff
@@ -112,16 +104,43 @@ classDiagram
 
 ```
 
-Note: You may make additional subclasses for commands other than `MAKE_MOVE` but they are not required
+The important thing is that along with the fields that are generally required for a `UserGameCommand`, that there is also an additional field named `move` that contains the fields of a `ChessMove` object in your JSON serialization.
```
[the remark this answers is not in the document]
Comment (leesjensen) [reply chess/6-gameplay/gameplay.md]: I reworked this a bit. Hopefully it is clearer now.

File: chess/6-gameplay/gameplay.md [hunk 3]
```diff
@@ -112,16 +104,43 @@ classDiagram
...
 | ---------------- | ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------- |
 | **LOAD_GAME**    | game (can be any type, just needs to be called `game`) | Used by the server to send the current game state to a client. When a client receives this message, it will redraw the chess board. |
 | **ERROR**        | String errorMessage                                    | This message is sent to a client when it sends an invalid command. The message must include the word `Error`.                       |
 | **NOTIFICATION** | String message                                         | This is a message meant to inform a player when another player made an action.                                                      |
 
+The following diagram represents a possible extensions of the `ServerMessage` class that you could use to support the different `ServerMessage` commands. You are free to follow this design, or come up with a representation of your own. Just make sure the the required fields are included in your JSON serialization.
```
[the remark this answers is not in the document]
Comment (leesjensen) [reply chess/6-gameplay/gameplay.md]: fixed

File: chess/6-gameplay/gameplay.md (hunk 1 above)
Comment (leesjensen) [reply chess/6-gameplay/gameplay.md]: Oh my. I obviously need some sleep. Thanks for catching this.

**Comment by frozenfrank** [comment]
Looks great @leesjensen! Thanks for making this happen.

File: chess/6-gameplay/gameplay.md (hunk 1 above)
Comment (frozenfrank) [reply chess/6-gameplay/gameplay.md]: You're doing great. Thanks for all your hard work 😄

**Pull request merged**